ament_index_python/ament_index_python/packages.py

```diff
@@ -36,7 +38,7 @@ def get_packages_with_prefixes():
     return get_resources('packages')
 
 
-def get_package_prefix(package_name):
+def get_package_prefix(package_name: str) -> Union[str, os.PathLike[str]]:
```

Comment:
Now that we've changed get_search_paths to return a list of strings, this can be:

```suggestion
def get_package_prefix(package_name: str) -> str
```